modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/parse/MBObjectParser.java

```diff
@@ -318,7 +326,7 @@ public String get(JSONArray json, int index) {
 
     /**
      * Quickly access required json tag (as a String).
-     * 
+     *
```

Comment:
lots of whitespace changes here, mask if this file has actually been modified

Reply:
I tried to revert the whitespace changes, that is annoying. I think I got all but one.